lib/tasks/webpacker/check_binstubs.rake

```diff
@@ -1,7 +1,8 @@
 namespace :webpacker do
   desc "Verifies that bin/webpack & bin/webpack-dev-server are present."
   task :check_binstubs do
-    unless File.exist?("bin/webpack") && File.exist?("bin/webpack-dev-server")
+    unless Webpacker.config.custom_compile? ||
+      (File.exist?("bin/webpack") && File.exist?("bin/webpack-dev-server"))
```

Comment:
@justin808 Yeah probably should test for `config/webpacker.yml` since that's more primary than checking binstubs. 

We discussed to remove logger output - don't think it's needed when assets are fresh.